packages/neon_framework/packages/talk_app/lib/src/blocs/room.dart

```diff
@@ -481,7 +481,7 @@ class _TalkRoomBloc extends InteractiveBloc implements TalkRoomBloc {
           b.clear();
 
           for (final entry in reactions.entries) {
-            if (entry.value.where((r) => r.actorId == account.username).isNotEmpty) {
+            if (entry.value.where((r) => r.actorId == account.credentials.loginName).isNotEmpty) {
```

Comment:
This is not going to work as the actorId is always a username and not a loginName (same for all the other changes in Talk).